### ISSUE TITLE
Better marketplace signposting

### ISSUE THREAD

**Issue by LottePitcher** [umbraco/OurUmbraco#825]
If you go to the Packages section on Our you get a nice big message telling you to search the marketplace. However, as @jacksorjacksor pointed out to me, if you google for a specific package chances are you are still going to be directed to the specific package page on the Our website, and specific package pages make no mention of the marketplace, so people would not know they are in the wrong place. Try searching for "umbraco clean starter kit".

Of course if you _were_ looking for pre v8 package info then you would be in the right place, so am not suggesting implementing any redirects. 

But how about a suitably worded banner at the top of all package pages? Which could include a link to the marketplace (searching for the package name). We could also manage a json file of the Our urls and the marketplace equivalents for the popular packages, so that we can put the correct link if it's listed in that json file. This json file feels a much easier approach than going anywhere near suggesting that a new textbox is added to the package doc type and giving people the ability to maintain their marketplace urls themselves...

I would appreciate HQ's thoughts please before I give this banner a go (I think I still have Our running locally!)

**Comment by nul800sebastiaan** [comment]
For now one could retire the package, for example:

https://our.umbraco.com/packages/developer-tools/cultivhangfire/
![image](https://github.com/umbraco/OurUmbraco/assets/304656/641a5593-0d64-4718-9a6d-d4c91330aaff)

In the package editing process you can leave a message, and apparently we allow links in the message:

![image](https://github.com/umbraco/OurUmbraco/assets/304656/cec2571e-cb47-47f4-8c1f-67dd47f7f66a)

**Comment by nul800sebastiaan** [comment]
Other than that, I'll have to get back to you, needs a think first.

**Comment by nul800sebastiaan** [comment]
Updated my first comment as I realized that retiring the package removes it from the search results completely.

**Comment by LottePitcher** [comment]
Yes, I just realised that myself when "retiring" my Admin Only Package.
https://our.umbraco.com/packages/backoffice-extensions/admin-only-property/

As there is no v7 version of this package, I'm ok with this not appearing in the search results. As I assume anyone searching for a package will have seen the existing banner about going to the marketplace instead.

**Comment by LottePitcher** [comment]
When viewing your own list of packages there is a misleading comment then for retired packages:

![image](https://github.com/umbraco/OurUmbraco/assets/4716542/ddd8cb6a-3633-410d-b918-4590ff3f5cf6)

Perhaps retired packages were removed from search results and this comment was not updated accordingly. 

I appreciate that the packages area of Our is not a priority but if Google search results are still directing people there perhaps it still has some importance.

Anyway, happy to help if contributions are wanted!

**Comment by LottePitcher** [comment]
Decided to re-save my package (thus un-retiring it) and add a comment at the top of the description re it's new home on the marketplace. Then it would still appear in searches and people get notification.

https://our.umbraco.com/packages/backoffice-extensions/admin-only-property/

![image](https://github.com/umbraco/OurUmbraco/assets/4716542/ae374a79-38a0-4d3b-bbd2-b34b37ee2def)
